server/src/main/java/io/druid/segment/realtime/appenderator/Appenderator.java

```diff
@@ -179,14 +161,11 @@ AppenderatorAddResult add(
    *
    * @return future that resolves when all pending data has been persisted, contains commit metadata for this persist
    */
-  default ListenableFuture<Object> persistAll(@Nullable Committer committer)
-  {
-    return persist(getSegments(), committer);
-  }
+  ListenableFuture<Object> persistAll(@Nullable Committer committer);
```

Comment:
Please also adjust the javadocs for other methods to stop referring to the "persist" method.

Reply:
Thanks. Updated javadoc.